### ISSUE TITLE
New submissions for Tue, 28 Nov 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#156]
# New submissions for Tue, 28 Nov 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### JWST and ALMA discern the assembly of structural and obscured components  in a high-redshift starburst galaxy
 - **Authors:** Zhaoxuan Liu, John D. Silverman, Emanuele Daddi, Annagrazia Puglisi, Alvio Renzini, Boris S. Kalita, Jeyhan S. Kartaltepe, Daichi Kashino, Giulia Rodighiero, Wiphu Rujopakarn, Tomoko L. Suzuki, Takumi S. Tanaka, Francesco Valentino, Irham Taufik Andika, Caitlin M. Casey, Andreas Faisst, Maximilien Franco, Ghassem Gozaliasl, Steven Gillman, Christopher C. Hayward, Anton M. Koekemoer, Vasily Kokorev, Erini Lambrides, Minju M. Lee, Georgios E. Magdis, Santosh Harish, Henry Joy McCracken, Jason Rhodes, Marko Shuntov
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2311.14809
 - **Pdf link:** https://arxiv.org/pdf/2311.14809
 - **Abstract**
 We present observations and analysis of the starburst, PACS-819, at z=1.45 ($M_*=10^{10.7}$ M$_{ \odot}$), using high-resolution ($0^{\prime \prime}.1$; 0.8 kpc) ALMA and multi-wavelength JWST images from the COSMOS-Web program. Dissimilar to HST/ACS images in the rest-frame UV, the redder NIRCam and MIRI images reveal a smooth central mass concentration and spiral-like features, atypical for such an intense starburst. Through dynamical modeling of the CO J=5--4 emission with ALMA, PACS-819 is rotation-dominated thus has a disk-like nature. However, kinematic anomalies in CO and asymmetric features in the bluer JWST bands (e.g., F150W) support a more disturbed nature likely due to interactions. The JWST imaging further enables us to map the distribution of stellar mass and dust attenuation, thus clarifying the relationships between different structural components, not discernable in the previous HST images. The CO J = 5 -- 4 and FIR dust continuum emission are co-spatial with a heavily-obscured starbursting core (<1 kpc) which is partially surrounded by much less obscured star-forming structures including a prominent arc, possibly a tidally-distorted dwarf galaxy, and a clump, either a sign of an ongoing violent disk instability or a recently accreted low-mass satellite. With spatially-resolved maps, we find a high molecular gas fraction in the central area reaching $\sim3$ ($M_{\text{gas}}$/$M_*$) and short depletion times ($M_{\text{gas}}/SFR\sim$ 120 Myrs) across the entire system. These observations provide insights into the complex nature of starbursts in the distant universe and underscore the wealth of complementary information from high-resolution observations with both ALMA and JWST.
#### Unveiling the 3D structure of nova shells with MUSE -- The case of RR  Pic
 - **Authors:** Lientur Celedón, Linda Schmidtobreick, Claus Tappert, Fernando Selman
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2311.14843
 - **Pdf link:** https://arxiv.org/pdf/2311.14843
 - **Abstract**
 Nova eruptions occur in cataclysmic variables when enough material has been accreted onto the surface of the white dwarf primary. As a consequence, the material that has been accumulated until then is expelled into the interstellar medium, forming an expanding nova shell around the system. Understanding the physical process that shapes the morphology of nova shells is essential to fully comprehend how the ejection mechanism operates during nova eruptions. Because of its closeness and age, the nova shell around the classical nova RR Pic (Nova Pic 1925) is an ideal target for studying the evolving morphology of nova shells. In this work, we present an IFS study of the RR Pic nova shell, with a particular emphasis on the extraction of the 3D morphology of the shell. The nova shell was observed by the Multi-Unit Spectroscopic Explorer (MUSE) instrument placed at the ESO-VLT. The MUSE datacube confirms the presence of the nova shell in H$\rm\alpha$, H$\rm\beta$ and [OIII], and very faintly in [NII]. A comparison with previous observations suggests that the shell continues in its free-expansion phase but with the different parts of the shell apparently expanding at different rates. The data analysis corroborates the previous vision that the shell is composed of an equatorial ring and polar filaments. At the same time, the new data also reveal that [OIII] is confined in gaps located in the tropical regions of the shell where no Hydrogen is observed. The flux measurements indicate that ~99% of the shell flux is confined to the equatorial ring, while the polar filaments show a flux asymmetry between the NE and SW filaments. We have estimated the mass of the shell to be ~5x10$^{-5}$M$_\odot$. From the analysis of the 3D-extracted data, we determine that the ring structure extends ~8,000 au from the central binary, and has a position angle of ~155 deg and an inclination of ~74 deg.
#### Discovery of a young, highly scattered pulsar PSR J1032-5804 with the  Australian SKA Pathfinder
 - **Authors:** Ziteng Wang, David L. Kaplan, Rahul Sengar, Emil Lenc, Andrew Zic, Akash Anumarlapudi, B. M. Gaensler, Natasha Hurley-Walker, Tara Murphy, Yuanming Wang
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2311.14880
 - **Pdf link:** https://arxiv.org/pdf/2311.14880
 - **Abstract**
 We report the discovery of a young, highly scattered pulsar in a search for highly circularly polarized radio sources as part of the Australian Square Kilometre Array Pathfinder (ASKAP) Variables and Slow Transients (VAST) survey. In follow-up observations with Murriyang/Parkes, we identified PSR J1032-5804 and measured a period of 78.7 ms, dispersion measure (DM) of 819$\pm$4 pc cm$^{-3}$, rotation measure of -2000$\pm$1 rad m$^{-2}$, and a characteristic age of 34.6 kyr. We found a pulse scattering timescale at 3 GHz of ~22 ms, implying a timescale at 1 GHz of ~3845 ms, which is the third most scattered pulsar known and explains its non-detection in previous pulsar surveys. We discuss the identification of a possible pulsar wind nebula and supernova remnant in the pulsar's local environment by analyzing the pulsar spectral energy distribution and the surrounding extended emission from multiwavelength images. Our result highlights the possibility of identifying extremely scattered pulsars from radio continuum images. Ongoing and future large-scale radio continuum surveys will offer us an unprecedented opportunity to find more extreme pulsars (e.g., highly scattered, highly intermittent, highly accelerated), which will enhance our understanding of the characteristics of pulsars and the interstellar medium.
#### Mapping a lower limit on the mass fraction of the cold neutral medium  using Fourier transformed HI 21cm emission line spectra: Application to the  DRAO Deep Field from DHIGLS and the HI4PI survey
 - **Authors:** Antoine Marchal, Peter G. Martin, Marc-Antoine Miville-Deschênes, Naomi M. McClure-Griffiths, Callum Lynn, Andrea Bracco, Luka Vujeva
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2311.15122
 - **Pdf link:** https://arxiv.org/pdf/2311.15122
 - **Abstract**
 We develop a new method for spatially mapping a lower limit on the mass fraction of the cold neutral medium by analyzing the amplitude structure of $\hat T_b(k_v)$, the Fourier transform of $T_b(v)$, the spectrum of the brightness temperature of HI 21cm line emission with respect to the radial velocity $v$. This advances a broader effort exploiting 21cm emission line data alone (without absorption line data, $\tau$) to extract integrated properties of the multiphase structure of the HI gas and to map each phase separately. Using toy models, we illustrate the origin of interference patterns seen in $\hat T_b(k_v)$. Building on this, a lower limit on the cold gas mass fraction is obtained from the amplitude of $\hat T_b$ at high $k_v$. Tested on a numerical simulation of thermally bi-stable turbulence, the lower limit from this method has a strong linear correlation with the "true" cold gas mass fraction from the simulation for relatively low cold gas mass fraction. At higher mass fraction, our lower limit is lower than the "true" value, because of a combination of interference and opacity effects. Comparison with absorption surveys shows a similar behavior, with a departure from linear correlation at $N_{\rm HI}\gtrsim 3-5\times10^{20}$ cm$^{-2}$. Application to the DRAO Deep Field (DF) from DHIGLS reveals a complex network of cold filaments in the Spider, an important structural property of the thermal condensation of the HI gas. Application to the HI4PI survey in the velocity range $-90 < v < 90$ km/s produces a full sky map of a lower limit on the mass fraction of the cold neutral medium at 16'.2 resolution. Our new method has the ability to extract a lower limit on the cold gas mass fraction for massive amounts of emission line data alone with low computing time and memory, pointing the way to new approaches suitable for the new generation of radio interferometers.
#### Universal relations in compact stars with exotic degrees of freedom
 - **Authors:** Manoj Kumar Ghosh, Anil Kumar, Pratik Thakur, Vivek Baruah Thapa, Monika Sinha
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); High Energy Physics - Phenomenology (hep-ph); Nuclear Theory (nucl-th)
 - **Arxiv link:** https://arxiv.org/abs/2311.15277
 - **Pdf link:** https://arxiv.org/pdf/2311.15277
 - **Abstract**
 The nature of the highly dense matter inside the supernova remnant compact star is not constrained by terrestrial experiment and hence is modeled phenomenologically to accommodate the astrophysical observations from compact stars as the observable properties of the compact stars are highly sensitive to the microscopic model of highly dense matter. However, there exists some universal relations between some macroscopic properties of compact stars independent of the matter model. We examine the universal relations for quantities moment of inertia - tidal love number - quadrupole moment. We also study some already established universal relations in non-radial oscillation frequencies with star compactness for the baryonic star with core composed of heavier baryons and for the hybrid star with core composed of strange quark matter in CFL phase surrounded by nucleonic matter. We find the hybrid star with core of quark matter in the CFL phase obeys the same universal relation as the hybrid star with normal quark matter. However, the baryonic star with strange and non-strange heavier baryons at the inner core fails to satisfy the universal relations.
#### Physics-Informed Neural Networks for Predicting the Asymptotic Outcome  of Fast Neutrino Flavor Conversions
 - **Authors:** Sajad Abbar, Meng-Ru Wu, Zewei Xiong
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2311.15656
 - **Pdf link:** https://arxiv.org/pdf/2311.15656
 - **Abstract**
 In the most extreme astrophysical environments, such as core-collapse supernovae (CCSNe) and neutron star mergers (NSMs), neutrinos can undergo fast flavor conversions (FFCs) on exceedingly short scales. Intensive simulations have demonstrated that FFCs can attain equilibrium states in certain models. In this study, we utilize physics-informed neural networks (PINNs) to predict the asymptotic outcomes of FFCs, by specifically targeting the first two moments of neutrino angular distributions. This makes our approach suitable for state-of-the-art CCSN and NSM simulations. Through effective feature engineering and the incorporation of customized loss functions that penalize discrepancies in the predicted total number of $\nu_e$ and $\bar\nu_e$, our PINNs demonstrate remarkable accuracies, with an error margin of $\lesssim3\%$. Our study represents a substantial leap forward in the potential incorporation of FFCs into simulations of CCSNe and NSMs, thereby enhancing our understanding of these extraordinary astrophysical events.
#### Empirical instability strip for classical Cepheids: I. The LMC galaxy
 - **Authors:** F. Espinoza-Arancibia, B. Pilecki, G. Pietrzyński, R. Smolec, P. Kervella
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2311.15849
 - **Pdf link:** https://arxiv.org/pdf/2311.15849
 - **Abstract**
 The instability strip (IS) of classical Cepheids has been extensively studied theoretically. Comparison of the theoretical IS edges with those obtained empirically, using the most recent Cepheids catalogs available, can provide us with insights into the physical processes that determine the position of the IS boundaries. In this study, we investigate the empirical positions of the IS of the classical Cepheids in the Large Magellanic Cloud (LMC), considering any effect that increases its width, to obtain intrinsic edges that can be compared with theoretical models. We use data of classical fundamental-mode (F) and first-overtone (1O) LMC Cepheids from the OGLE-IV variable star catalog, together with a recent high-resolution reddening map from the literature. Our final sample includes 2058 F and 1387 1O Cepheids. We studied their position on the Hertzsprung-Russell diagram and determined the IS borders by tracing the edges of the color distribution along the strip. We obtain the blue and red edges of the IS in V- and I-photometric bands, in addition to $\log T_{\rm eff}$ and $\log L$. The results obtained show a break located at the Cepheids' period of about 3 days, which was not reported before. We compare our empirical borders with theoretical ones published in the literature obtaining a good agreement for specific parameter sets. The break in the IS borders is most likely explained by the depopulation of second and third crossing classical Cepheids in the faint part of the IS, since blue loops of evolutionary tracks in this mass range do not extend blueward enough to cross the IS at the LMC metallicity. Results from the comparison of our empirical borders with theoretical ones prove that our empirical IS is a useful tool for constraining theoretical models.
#### A First Look with JWST Aperture Masking Interferometry (AMI): Resolving  Circumstellar Dust around the Wolf-Rayet Binary WR 137 beyond the Rayleigh  Limit
 - **Authors:** Ryan M. Lau, Matthew J. Hankins, Joel Sanchez-Bermudez, Deepashri Thatte, Anthony Soulain, Rachel A. Cooper, Anand Sivaramakrishnan, Michael F. Corcoran, Alexandra Z. Greenbaum, Theodore R. Gull, Yinuo Han, Olivia C. Jones, Thomas Madura, Anthony F. J. Moffat, Mark R. Morris, Takashi Onaka, Christopher M. P. Russell, Noel D. Richardson, Nathan Smith, Peter Tuthill, Kevin Volk, Gerd Weigelt, Peredur M. Williams
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2311.15948
 - **Pdf link:** https://arxiv.org/pdf/2311.15948
 - **Abstract**
 We present infrared aperture masking interferometry (AMI) observations of newly formed dust from the colliding winds of the massive binary system Wolf-Rayet (WR) 137 with JWST using the Near Infrared Imager and Slitless Spectrograph (NIRISS). NIRISS AMI observations of WR 137 and a point-spread-function calibrator star, HD~228337, were taken using the F380M and F480M filters in 2022 July and August as part of the Director's Discretionary Early Release Science (DD-ERS) program 1349. Interferometric observables (squared visibilities and closure phases) from the WR 137 "interferogram" were extracted and calibrated using three independent software tools: ImPlaneIA, AMICAL, and SAMpip. The analysis of the calibrated observables yielded consistent values except for slightly discrepant closure phases measured by ImPlaneIA. Based on all three sets of calibrated observables, images were reconstructed using three independent software tools: BSMEM, IRBis, and SQUEEZE. All reconstructed image combinations generated consistent images in both F380M and F480M filters. The reconstructed images of WR 137 reveal a bright central core with a $\sim300$ mas linear filament extending to the northwest. A geometric colliding-wind model with dust production constrained to the orbital plane of the binary system and enhanced as the system approaches periapsis provided a general agreement with the interferometric observables and reconstructed images. Based on a colliding-wind dust condensation analysis, we suggest that dust formation within the orbital plane of WR 137 is induced by enhanced equatorial mass-loss from the rapidly rotating O9 companion star, whose axis of rotation is aligned with that of the orbit.

by olozhika (Xing Yuchen). 

2023-11-28
